doc/getting-started/installation.rst

```diff
@@ -247,10 +246,101 @@ Reconnect to the server with port-forwarding, and start Jupyter:
 
     ssh -i "your-ssh-key" your-user-name@your-server-ip -L 8888:localhost:8888
     conda activate vtk_env
-    jupyter notebook --NotebookApp.token='' --no-browser --port=8888
+    jupyter lab --NotebookApp.token='' --no-browser --port=8888
 
 Visit ``localhost:8888`` in the web browser.
 
+Running on WSL
+~~~~~~~~~~~~~~
+Similar to the example of the remote server above, the windows subsystem for Linux does
+not provide an x-server for visualization. Instead, the fastest way to get up and
+running on WSL is through ``jupyter lab``.
+
+First, make sure you have installed the correct environment through Miniconda and
+related packages:
+
+.. code-block:: bash
+
+    wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
+    bash miniconda.sh -b -p miniconda
+    echo '. $HOME/miniconda/etc/profile.d/conda.sh' >> ~/.bashrc && source ~/.bashrc
+    conda create --name vtk_env python=3.9
+    conda activate vtk_env
+    conda install nodejs  # required when importing pyvista in Jupyter
+    pip install jupyter pyvista ipyvtklink
+
+    # To avoid "ModuleNotFoundError: No module named 'vtkOpenGLKitPython' " when importing vtk
+    # https://stackoverflow.com/q/32389599
+    # https://askubuntu.com/q/629692
+    sudo apt update && sudo apt install python-qt4 libgl1-mesa-glx
+
+VTK Link to Jupyter
+^^^^^^^^^^^^^^^^^^^
+There are two ways to get vtk rendering 3D objects in ``jupyter lab``. First you
```

Comment:
It's up to debate, but I'm following the formatting used in their installation docs at https://jupyterlab.readthedocs.io/en/stable/getting_started/installation.html

```suggestion
There are two ways to get vtk rendering 3D objects in JupyterLab. First you
```